rtos/TARGET_CORTEX/mbed_rtos_storage.h

```diff
@@ -40,17 +40,17 @@ extern "C" {
  implementation specific, header file, therefore limiting scope of possible changes.
  */
 
-#include "rtx_lib.h"
+#include "rtx_os.h"
 #include "mbed_rtx_conf.h"
 
-typedef os_mutex_t mbed_rtos_storage_mutex_t;
-typedef os_semaphore_t mbed_rtos_storage_semaphore_t;
-typedef os_thread_t mbed_rtos_storage_thread_t;
-typedef os_memory_pool_t mbed_rtos_storage_mem_pool_t;
-typedef os_message_queue_t mbed_rtos_storage_msg_queue_t;
-typedef os_event_flags_t mbed_rtos_storage_event_flags_t;
-typedef os_message_t mbed_rtos_storage_message_t;
-typedef os_timer_t mbed_rtos_storage_timer_t;
+typedef osRtxMutex_t mbed_rtos_storage_mutex_t;
```

Comment:
I'm not sure about this, isn't the `os_*_t` more external type then `osRtx*_t`? @JonatanAntoni

Reply:
No, os_*_t are internal defines, only. The public but RTX5 implementation specific typedefs are osRtx*_t, defined in rtx_os.h. The CMSIS-RTOS2 API typedefs are os*Id_t, defined in cmsis_os2.h. The latter one are only void*, so those can't be used if you need the control block sizes.